src/parallel.cpp

```diff
@@ -617,54 +626,51 @@ void HandleBlockMessageThread(CNode *pfrom, const string strCommand, const CBloc
 }
 
 
-CCheckQueue<CScriptCheck> *CAllScriptCheckQueues::GetScriptCheckQueue()
+// for newly mined block validation, return the first queue not in use.
+CCheckQueue<CScriptCheck> *CParallelValidation::GetScriptCheckQueue()
 {
-    // for newly mined block validation, return the first queue not in use.
-    CCheckQueue<CScriptCheck> *pqueue = NULL;
-    if (Size() > 0)
+    while (true)
     {
```

Comment:
just as a note: if you're going to indent code , please don't change internals of the code at the same time...in the future put indentation and/or moving of big chucks of code in their own commit.